server/userNotification/model.ts

```diff
@@ -45,7 +52,7 @@ export class UserNotification extends Model<
 	@Column(DataType.BOOLEAN)
 	manuallySetIsRead!: CreationOptional<boolean>;
 
-	@BelongsTo(() => ActivityItem, {
+	@BelongsTo(() => ActivityItem<InsertableActivityItem>, {
```

Comment:
This typing leaves something to be desired, leads to some annoying types. I think UserNotification can only have like `pub-commented` or `review-commented` or something, so this could be more precise.